python/.dockerignore

```diff
@@ -1,9 +1,10 @@
 *.git
 **/*.pyc
 **/__pycache__
-./python/data
-./python/models
-./python/logs
-target/
+./data/
+./models/
+./logs/
+./env/
+./target/
 **/test_output/
 **/test_output/
```

Comment:
[Extremely minor] Looks like we have a duplicated line here, can we remove one of the `**/test_output/` lines?

Reply:
Good catch.